config/local/config.yml

```diff
@@ -1,277 +1,284 @@
+        critical_threshold_wei: <critical_threshold_wei> # e.g. 580000000000000000
+      gas_price: <gas_price> # e.g. "30000000000"
+      min_gas_tip_cap: <min_gas_tip_cap> # e.g. 30000000000
+
   optimism:
     chain_name: "optimism"
     chain_id: "10"
     type: evm
     environment: "mainnet"
     gas_token_symbol: "ETH"
-    gas_token_coingecko_id: "ethereum"
     gas_token_decimals: 18
+    num_block_confirmations_before_fill: <num_block_confirmations_before_fill> # e.g. 1
+    batch_uusdc_settle_up_threshold: <batch_uusdc_settle_up_threshold> # e.g. 5000000
+    min_fill_size: <min_fill_size> # e.g. 5000000
+    max_fill_size: <max_fill_size> # e.g. 10000000000
+    max_rebalancing_gas_threshold: <max_rebalancing_gas_threshold> # e.g. 1000000
+    min_fee_bps: <min_fee_bps> # e.g. 100
+    solver_address: <solver_address> # e.g. "0x8EB49E3D65d74967CC0Fe987FA2d015ae816352E"
     evm:
-      rpc: <optimism_rpc_server_url>
+      rpc: <optimism_rpc_server_url> # e.g. "https://mainnet.optimism.io"
       rpc_basic_auth_var: <server_password>
       contracts:
         usdc_erc20_address: 0x0b2c639c533813f4aa9d7837caf62653d097ff85
       signer_gas_balance:
-        warning_threshold_wei: 180000000000000000
-        critical_threshold_wei: 60000000000000000
-  optimism-testnet:
-    chain_name: "optimism"
-    chain_id: "11155420"
-    type: evm
-    environment: "testnet"
-    gas_token_symbol: "ETH"
-    gas_token_decimals: 18
-    evm:
-      rpc: <optimism_testnet_rpc_server_url>
-      rpc_basic_auth_var: <server_password>
-      contracts:
-        usdc_erc20_address: 0x5fd84259d66Cd46123540766Be93DFE6D43130D7
-      signer_gas_balance:
-        warning_threshold_wei: 250000000000000000
-        critical_threshold_wei: 0
+        warning_threshold_wei: <warning_threshold_wei> # e.g. 180000000000000000
+        critical_threshold_wei: <critical_threshold_wei> # e.g. 60000000000000000
+      gas_price: <gas_price> # e.g. "30000000000"
+      min_gas_tip_cap: <min_gas_tip_cap> # e.g. 30000000000
+
   arbitrum:
     chain_name: "arbitrum"
     chain_id: "42161"
     type: evm
     environment: "mainnet"
     gas_token_symbol: "ETH"
-    gas_token_coingecko_id: "ethereum"
     gas_token_decimals: 18
     quick_start_num_blocks_back: 300000
     fast_transfer_contract_address: "0xD6F9F9E2C231E023fe0A8d752bc4080a112a1EbA"
-    solver_address: "0x8EB49E3D65d74967CC0Fe987FA2d015ae816352E"
+    solver_address: <solver_address> # e.g. "0x8EB49E3D65d74967CC0Fe987FA2d015ae816352E"
     hyperlane_domain: "42161"
+    num_block_confirmations_before_fill: <num_block_confirmations_before_fill> # e.g. 1
     batch_uusdc_settle_up_threshold: 5000000
+    min_fill_size: <min_fill_size> # e.g. 5000000
+    max_fill_size: <max_fill_size> # e.g. 10000000000
+    max_rebalancing_gas_threshold: <max_rebalancing_gas_threshold> # e.g. 1000000
+    min_fee_bps: <min_fee_bps> # e.g. 100
     evm:
```

Comment:
its a little confusing to me that this evm: category is contained within the arbitrum category

Reply:
but dont think  this is a concern for this PR

Reply:
This field contains any specific evm config for all EVM chains, thats why it is included for Arbitrum, Optimism, etc. (any evm chains, similar to how the cosmos field is used in Noble and other cosmos chains configured)

https://github.com/skip-mev/skip-go-fast-solver/pull/12/files#diff-f7c7c70e0668978ba8e2e9df80ef8e6521d67f305080bb0a7971e2783e52c7b4R78